packages/epics/src/agreements/plugins/deploy-funds/validation.ts

```diff
@@ -0,0 +1,18 @@
+import { z } from 'zod';
+
+const ETH_ADDRESS_REGEX = /^0x[a-fA-F0-9]{40}$/;
+
+export const schemaPayForExpenses = z.object({
+  recipient: z
+    .string()
+    .min(1, { message: 'Recipient is required' })
+    .regex(ETH_ADDRESS_REGEX, { message: 'Invalid Ethereum address' }),
+  payouts: z
```

Comment:
we duplicate those validation objects. Let's move the `recipient` validation object next to the `recipient-field.tsx` and reuse. I'm pushing a commit to illustrate